ext/drbbdup/drbbdup.h

```diff
@@ -64,10 +64,11 @@ typedef enum {
     DRBBDUP_ERROR_INVALID_OPND,            /**< Operation failed: invalid case opnd. */
     DRBBDUP_ERROR_CASE_ALREADY_REGISTERED, /**< Operation failed: already registered. */
     DRBBDUP_ERROR_CASE_LIMIT_REACHED,      /**< Operation failed: case limit reached. */
-    DRBBDUP_ERROR_ALREADY_INITIALISED,     /**< DRBBDUP can only be initialised once. */
-    DRBBDUP_ERROR,                         /**< Operation failed. */
-    DRBBDUP_ERROR_UNSET_FEATURE,           /**< Operation failed: feature not set. */
-    DRBBDUP_ERROR_NOT_INITIALIZED,         /**< Operation failed: not initialized. */
+    DRBBDUP_ERROR_ALREADY_INITIALISED, /**< DRBBDUP can only be initialised once. This is
```

Comment:
The comment indentation seems off. But the tests didn't complain, so i assume this is clang.

Reply:
Yeah, it is clang-format.

Reply:
Note that officially we would prefer to move this to a whole-line beforehand comment to make it "prettier"; the same for the comment on line 1758 of drbbdup.c where officially we prefer the `*/` on its own line.  But since there is no way to make clang-format enforce that I've given up on that as a style rule.